runtime/parachains/src/paras_inherent/benchmarking.rs

```diff
@@ -60,7 +60,7 @@ benchmarks! {
 				.collect();
 
 		let scenario = BenchBuilder::<T>::new()
-			.build(cores_with_backed, &[1], None);
+			.build(cores_with_backed, &[], None);
```

Comment:
I would be nice to remove the cryptic slice `&[]` of integers from the `build` param and add additional builder methods.

Reply:
4a4d03c